orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/entity/OrchSessionItem.java

```diff
@@ -8,9 +8,18 @@
 public class OrchSessionItem {
 
     public static final String ATTRIBUTE_SESSION_ID = "SessionId";
+    public static final String ATTRIBUTE_IS_NEW_ACCOUNT = "IsNewAccount";
+
+    public enum AccountState {
```

Comment:
Should this exist as a separate class in shared/entity?

Reply:
Not really sure we'd reuse this anywhere else apart from _session-y_ contexts, so not sure what benefit we'd get from separating it out

Reply:
fair enough!